source/seedsource/views.py

```diff
@@ -14,10 +14,12 @@
 
 from seedsource.models import TransferLimit, SeedZone, RunConfiguration, Region
 from seedsource.report import Report
-from seedsource.serializers import RunConfigurationSerializer, SeedZoneSerializer, GeneratePDFSerializer
+from seedsource.serializers import RunConfigurationSerializer, SeedZoneSerializer, GenerateReportSerializer
 from seedsource.serializers import TransferLimitSerializer, RegionSerializer
 from seedsource.utils import get_elevation_at_point, get_regions_for_point
 
+PPTX_MIME_TYPE = 'application/vnd.openxmlformats-officedocument.presentationml.presentation' # maybe we need it, maybe we dont
```

Comment:
I'm testing this with the React component. It may end up on the cutting room floor.